packages/pds/src/db/migrations/20230529T222706121Z-suggested-follows.ts

```diff
@@ -0,0 +1,12 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('suggested_follow')
+    .addColumn('did', 'varchar', (col) => col.primaryKey())
```

Comment:
Any use for a column here to control the order?

Reply:
actually yeah that's probably smart 👍 otherwise we just get them in did-alphabetic order